### PR TITLE
Cdd msk vocab feature

### PR DESCRIPTION
**Pull request by sheridancbio**

### CODE REVIEW

File: src/main/java/org/cbioportal/cdd/model/MSKClinicalAttributeMetadata.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+import io.swagger.annotations.ApiModelProperty;
+import java.io.Serializable;
+import java.util.*;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * @author Avery Wang, Manda Wilson
+ */
+//@JsonInclude(Include.NON_NULL)
```
Comment (averyniceday) [inline src/main/java/org/cbioportal/cdd/model/MSKClinicalAttributeMetadata.java:35]: can remove this

File: src/main/java/org/cbioportal/cdd/model/MSKClinicalAttributeMetadata.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    public String getDataType() {
+        return dataType;
+    }
+
+    @JsonProperty("typeLabel")
+    public void setDataType(String dataType) {
+        this.dataType = dataType;
+    }
+
+    @JsonProperty("blah")
```
Comment (averyniceday) [inline src/main/java/org/cbioportal/cdd/model/MSKClinicalAttributeMetadata.java:104]: probably shouldn't be blah 😆 also we could. add the standard comments if we wanted

File: src/main/java/org/cbioportal/cdd/model/MSKVocabResponse.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import io.swagger.annotations.ApiModelProperty;
+import java.io.Serializable;
+import java.util.*;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * @author Avery Wang, Manda Wilson
+ */
+//@JsonInclude(Include.NON_NULL)
```
Comment (averyniceday) [inline src/main/java/org/cbioportal/cdd/model/MSKVocabResponse.java:35]: remove

File: src/main/java/org/cbioportal/cdd/repository/topbraid/ClinicalAttributeMetadataRepositoryConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    @Qualifier("topBraidSessionConfiguration")
+    public TopBraidSessionConfiguration topBraidSessionConfiguration() {
+        TopBraidSessionConfiguration topBraidSessionConfiguration = new TopBraidSessionConfiguration(); 
+        topBraidSessionConfiguration.setURL(topbraidUrl);
+        topBraidSessionConfiguration.setUsername(topbraidUsername);
+        topBraidSessionConfiguration.setPassword(topbraidPassword);
+        return topBraidSessionConfiguration;
+    }
+
+    @Bean
```
Comment (averyniceday) [inline src/main/java/org/cbioportal/cdd/repository/topbraid/ClinicalAttributeMetadataRepositoryConfiguration.java:61]: actually this looks like it's missing a commit?

**Comment by sheridancbio** [comment]
changes incorporated into #120

**Pull request closed without merging**